pkg/operator/operator.go

```diff
@@ -156,8 +155,13 @@ func (o *Operator) WithControllers(ctx context.Context, controllers ...corecontr
 func (o *Operator) WithWebhooks(ctx context.Context, webhooks ...knativeinjection.ControllerConstructor) *Operator {
 	if !injection.GetOptions(ctx).DisableWebhook {
 		o.webhooks = append(o.webhooks, webhooks...)
-		lo.Must0(o.Manager.AddReadyzCheck("webhooks", knativeChecker("readiness")))
-		lo.Must0(o.Manager.AddHealthzCheck("webhooks", knativeChecker("health")))
+		go func(ctx context.Context) {
```

Comment:
Do we need some backoff here? Does this just aggressively spam the "waiting" log line?